code/controllers/subsystem/radio.dm

```diff
@@ -14,6 +14,7 @@ SUBSYSTEM_DEF(radio)
 	"Special Ops" 	= DTH_FREQ,
 	"Syndicate" 	= SYND_FREQ,
 	"SyndTeam" 		= SYNDTEAM_FREQ,
+	"Soviet"		= SOV_FREQ,
```

Comment:
Ok. Ill bite.


Why. Just why. Why do they need this? TSF doesn't have one, and there isn't "NT Radio" per say, its split into multiple departments. Wouldn't the soviets do the same?

Reply:
@AffectedArc07 the idea is that Paradise doesn't have any "third faction" / "neutral" force for the admins to use yet.  

TSF is NT aligned and uses their channels, the syndicate is NT hostile and has their own channel. I thought it would make sense for the Soviets who are neither NT nor TSF aligned to have their own channel.

However I am open to handling this a different way if it doesn't make sense.